### PR TITLE
Change ability button text for some roles

### PR DESCRIPTION
**Pull request by PEPPERcula**

### CODE REVIEW

**Comment by NikoCat233** [comment]
String too long

File: Resources/Lang/en_US.json [hunk 1]
```diff
@@ -526,7 +527,7 @@
   "PacifistInfo": "Vent to reset kill cooldowns",
   "RebirthInfo": "Arise Again",
   "MonarchInfo": "Give your crew extra voting power!",
-  "AbyssbringerInfo": "Create Black Holes",
+  "AbyssbringerInfo": "Place Black Holes",
```
Comment (NikoCat233) [inline Resources/Lang/en_US.json:530]: too long, did you check in game yourself?

**Review by NikoCat233 — CHANGES_REQUESTED** [review]

File: Resources/Lang/en_US.json (hunk 1 above)
Comment (PEPPERcula) [reply Resources/Lang/en_US.json:530]: ? im confusfed why you think it's too long bc there is more that are longer than that lol

**Comment by NikoCat233** [comment]
build and play yourself
share a screenshoot lol

**Comment by PEPPERcula** [comment]
image

**Comment by NikoCat233** [comment]
Think a cool one word string

**Pull request merged**